packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/en.ftl

```diff
@@ -2,6 +2,7 @@
 ## User followed a password reset link and is now prompted to create a new password
 
 complete-reset-pw-header = Create new password
```

Comment:
Some of the strings are no longer in use and should be safe to remove.
- complete-reset-pw-header
- complete-reset-password-warning-message-2
- complete-reset-password-recovery-key-error-v2
- complete-reset-password-recovery-key-link

Reply:
If they're not used anymore, then agree it'd be great to have these removed.

Reply:
Couldn't remember if we keep them around for a bit or not.